lib/charms/ovn_charm.py

```diff
@@ -322,15 +322,9 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         if self.enable_openstack:
             metadata_agent = 'neutron-ovn-metadata-agent'
-            # TODO: replace with ussuri ``neutron-ovn-metadata-agent`` pkg
-            self.packages.extend(['python3-neutron', 'haproxy'])
+            self.packages.extend([metadata_agent])
             self.services.append(metadata_agent)
```

Comment:
This looks like we are adding metadata_agent tiwce.

Reply:
We need to add it both to the packages to install and the services to look after